Tests/StreamChatUITests/SnapshotVariant.swift

```diff
@@ -44,3 +48,13 @@ enum SnapshotVariant: String, Hashable, CaseIterable {
         }
     }
 }
+
+extension Array where Element == SnapshotVariant {
+    static var allCases: [SnapshotVariant] {
+        SnapshotVariant.allCases
+    }
+    
+    static var lightAndDarkCases: [SnapshotVariant] {
```

Comment:
Nice, rename this to `userInterfaceStyleCases` and use it as the default value for snapshots, and I'm happy :)